packages/web/src/components/range/RangeInput.js

```diff
@@ -33,15 +33,6 @@ class RangeInput extends Component {
 		};
 	}
 
-	componentDidMount() {
```

Comment:
@mohdashraf010897 Why did you remove this?

Reply:
done